src/phoenix/server/api/types/Span.py

```diff
@@ -101,9 +110,14 @@ def from_dict(
         )
 
 
+@strawberry.type
+class SpanAsExample(Example): ...
```

Comment:
Do we want this in a separate file with the other types?

Reply:
This is only relevant to this file so I think it's okay. I can move the interface out if needed.